prospectmagazine.co.uk.txt

```diff
@@ -1,26 +1,9 @@
-#basics
-author: (//div[contains(@class,'author')])[1]
-date: substring-before(//a[@class='issue'], '&mdash;')
-#body://div[@class = 'entry']
-# use this until move_into support is ready
-body: //div[@class = 'entry' or @class='standfirst' or @class='lead_image']
+body: //div[contains(concat(' ',normalize-space(@id),' '),' post_content ')]
 
-#moves header image and tagline into body
-move_into(//div[@class='entry']/div)://div[@class = 'lead_image']
-move_into(//div[@class='entry']/div)://div[@class = 'standfirst']
+requires_login: yes
+login_uri: https://www.prospectmagazine.co.uk/wordpress/wp-login.php
```

Comment:
Are you sure about that?
Looks like the login form in the frontpage goes to `https://www.prospectmagazine.co.uk/` only.

Reply:
Yes, i am sure. The site is a Wordpress, the 'standalone' login page is this URL.

Reply:
Okay